elements/pfe-tabs/BaseTabs.ts

```diff
@@ -298,9 +296,11 @@ export abstract class BaseTabs extends LitElement {
     }
   }
 
-  #select(selectedTab: BaseTab): void {
+  async #select(selectedTab: BaseTab): Promise<void> {
     this.#activate(selectedTab);
     this.#focusTab = selectedTab;
+    await this.updateComplete;
+    this.#focusTab.focus();
```

Comment:
LGTM. Reserving this comment for @nikkimk 's review

Reply:
Looks okay to me.